tests/unit/Cache/Adapter/Libmemcached/GetKeysCest.php

```diff
@@ -39,6 +39,17 @@ public function cacheAdapterLibmemcachedGetKeys(UnitTester $I)
             getOptionsLibmemcached()
         );
 
+        $versions = $adapter->getAdapter()->getVersion();
+
+        foreach ($versions as $server => $version) {
+            // https://www.php.net/manual/en/memcached.getallkeys.php#123793
+            if (version_compare($version, '1.4.23', '>=')) {
```

Comment:
As I can see it is fixed https://bugs.launchpad.net/libmemcached/+bug/1534062
Could we compare more specific version range  here?

Reply:
Also add please launchpad link too,